desktop/components/main_layout/templates/index.jade

```diff
@@ -17,7 +17,7 @@ html(
       
     //- Don't include fonts if Reflection is requesting it		
     if sd.BROWSER && sd.BROWSER.family != 'PhantomJS'
-      link( type='text/css', rel='stylesheet', href='#{sd.WEBFONT_URL}/force-webfonts.css')
+      link( type='text/css', rel='stylesheet', href='#{sd.WEBFONT_URL}force-webfonts.css')
```

Comment:
Works for me, but this trailing slash could probably just be removed in the environment variable?